Tests/Fixtures/Etalon/annotated_conditional_controller.yml

```diff
@@ -1,86 +1,91 @@
 get_users:
   path:      /users.{_format}
   controller:   ::getUsersAction
-  requirements: {_method: GET}
+  method: GET
 
 get_user:
   path:      /users/{slug}.{_format}
   controller:   ::getUserAction
-  requirements: {_method: GET, slug: '[a-z]+'}
+  requirements: {slug: '[a-z]+'}
+  method: GET
 
 patch_users:
   path:      /users.{_format}
   controller:   ::patchUsersAction
-  requirements: {_method: PATCH}
+  method: PATCH
 
 patch_user:
   path:      /users/{slug}.{_format}
   controller:   ::patchUserAction
-  requirements: {_method: PATCH, slug: '[a-z]+'}
+  requirements: {slug: '[a-z]+'}
+  method: PATCH
 
 get_user_comment:
   path:      /users/{slug}/comments/{id}.{_format}
   controller:   ::getUserCommentAction
-  requirements: {_method: GET, slug: '[a-z]+', id: '\d+'}
+  requirements: {slug: '[a-z]+', id: '\d+'}
+  method: GET
 
 rate_user:
   path:      /users/{slug}/rate.{_format}
   controller:   ::rateUserAction
-  requirements: {_method: POST, slug: '[a-z]+'}
+  requirements: {slug: '[a-z]+'}
+  method: POST
 
 rate_user_comment:
   path:      /users/{slug}/rate_comment/{id}.{_format}
   controller:   ::rateUserCommentAction
-  requirements: {_method: PATCH, slug: '[a-z]+', id: '\d+'}
+  requirements: {slug: '[a-z]+', id: '\d+'}
+  method: PATCH
 
 bget_user:
   path:      /users/{slug}/bget.{_format}
   controller:   ::bgetUserAction
-  requirements: {_method: GET}
+  method: GET
 
 bpost_user:
   path:      /users/{slug}/bpost.{_format}
   controller:   ::bpostUserAction
-  requirements: {_method: POST}
+  method: POST
```

Comment:
Yep, I know, but the other test fixtures use it like this. I think fixing that should be another PR. Or should I change it here?